src/index.js

```diff
@@ -1,74 +1,89 @@
-const {app, BrowserWindow} = require('electron');
+const { app, BrowserWindow } = require('electron');
 const path = require('path');
 const url = require('url');
 const Main = require('./main/main.js');
+const DataStore = require('./main/data-store/data-store');
 
 // Keep a global reference of the window object, if you don't, the window will
 // be closed automatically when the JavaScript object is garbage collected.
-let win
+let win;
```

Comment:
Did you run through `prettier` to get all these format updates? Not for this PR, but at some point we should add a prettier hook like the idyll repo has to help with formatting. https://github.com/idyll-lang/idyll/blob/master/package.json#L24

Reply:
Ah I think my format on save local settings must've gotten in the way haha. And I agree! I can add that in the next PR if it's not going to be pulled in sooner than that